Makefile

```diff
@@ -615,36 +620,6 @@ lint : $(SOURCES)
 	| $(GREP) -v "github.com/git-lfs/git-lfs" \
 	| $(GREP) "."
 
-# generate index.txt for ronn HTML man page link generation
-.PHONY : index.txt
-index.txt :
-	@echo "Generating index.txt for ronn"
-	@( \
-		printf "# internal\n" >index.txt; \
-		for f in $$(cd docs/man && ls git-lfs*.ronn); do \
-			l=$$(printf "$$f" | sed -E 's/\.([1-9])\.ronn$$/(\1)/'); \
-			printf "%s %s\n" "$$l $$f"; \
-		done >>index.txt; \
```

Comment:
I think there's a couple of [lines](https://github.com/git-lfs/git-lfs/blob/5521dc13bc0ae75d45ab7c373fdaa7fdfc893607/.gitignore#L6-L8) in `.gitignore` which could profitably be removed since `index.txt` is no longer generated for ronn:
```
# only allow man/*.\d.ronn files
man/*
index.txt
```

These could just be:
```
man/*
```